tensorflow/core/grappler/optimizers/auto_mixed_precision_lists.h

```diff
@@ -132,6 +132,7 @@ class AutoMixedPrecisionListsCuda : public AutoMixedPrecisionLists {
         "LSTMBlockCell",
         "LSTMBlockCellGrad",
         "MatMul",
+        "TRTEngineOp"
```

Comment:
Can you explain why we want to add this node to the allowedlist, when the pass is run before TF-TRT conversion?